index.html

```diff
@@ -0,0 +1,236 @@
+                                <li>Bootstrap</li>
+                                <li>Ruby</li>
+                            </ul>
+                        </div>
+                        <button class="action-btn">See Project</button>
+                    </div>
+                </div>
+                <div class="card-container">
+                    <div class="card-item">
+                        <div class="right-block">
+                            <h3 class="card">Profesional Art Printing Data</h3>
+                            <p class="card">A daily selection of privately personalized reads; no accounts or sign-ups
+                                required. has been the industry's standard dummy text ever since the 1500s, when an unknown
+                                printer took a standard dummy text.</p>
+                            <ul class="tags">
+                                <li>html</li>
+                                <li>Bootstrap</li>
+                                <li>Ruby</li>
+                            </ul>
+                        </div>
+                        <button class="action-btn">See Project</button>
+                    </div>
+                </div>
+                <div class="card-container">
+                    <div class="card-item">
+                        <div class="right-block">
+                            <h3 class="card">Profesional Art Printing Data</h3>
+                            <p class="card">A daily selection of privately personalized reads; no accounts or sign-ups
+                                required. has been the industry's standard dummy text ever since the 1500s, when an unknown
+                                printer took a standard dummy text.</p>
+                            <ul class="tags">
+                                <li>html</li>
+                                <li>Bootstrap</li>
+                                <li>Ruby</li>
+                            </ul>
+                        </div>
+                        <button class="action-btn">See Project</button>
+                    </div>
+                </div>
+    
+            </section>
+            <section class="about-me">
+                <div class="the-resume">
+                  <div class="myresume">
+                    <h3 class="title-h3">About me</h3>
+                    <p class="about-paragraph">
+                      Hello I’m a software developer! I can help you build a product ,
+                      feature or website Look through some of my work and experience! If
+                      you like what you see and have a project you need coded, don’t
+                      hestiate to contact me.
+                    </p>
+                    <button class="resume" type="button">Get My Resume</button>
+                  </div>
+                </div>
+                <di class="vector">
+                    <img src="./images/Vector 4.svg" alt="line" />
+                </di>
```

Comment:
- [ ] Here I assume you wanted to use `<div>` tag to wrap the image, but as you can see it's only `<di>` now 🤔. I suggest you make it `<div>` since this is semantically correct and it will help screen readers 🤖.